app/src/main/java/com/nextcloud/talk/adapters/messages/IncomingTextMessageViewHolder.kt

```diff
@@ -103,10 +104,26 @@ class IncomingTextMessageViewHolder(itemView: View, payload: Any) :
                 true,
                 viewThemeUtils
             )
+
+            val spansFromString: Array<Any> = processedMessageText!!.getSpans(
+                0,
+                processedMessageText.length,
+                Any::class.java
+            )
+
+            if (spansFromString.isNotEmpty()) {
+                binding.bubble.layoutParams.apply {
+                    width = FlexboxLayout.LayoutParams.MATCH_PARENT
+                }
+                binding.messageText.layoutParams.apply {
+                    width = FlexboxLayout.LayoutParams.MATCH_PARENT
+                }
+            }
```

Comment:
this will cause that the width is not reset for other bubbles. So sending some random short text after sending a table, the width is still the one of the table.

Adding an else which sets the width to WRAP_CONTENT should solve it.